doc/Architecture.md

```diff
@@ -279,7 +279,81 @@ The offchain code that constructs the DeNS database will always treat the _most
 
 ## Architecture - Arweave (Technical) (WIP)
 
-TODO
+It seems that the maximum amount of data we can upload paying the minimum amount possible is:
+
+```bash
+https://arweave.net/price/256000
+212017846
+```
+
+This price is given in Wilson, where a Wilson is related to Ar (the arweave currency) as:
+
+```python
+Ar = 1e12 wilson
+```
+
+At today the price of an Ar is around `9.5` usd, this means that the minimum cost of a transaction is : `0.002014169537` usd.
+And the maximum size of a transaction with such a price is above 250 KB.
+
+Additionally some gateways allows the upload of certain amount of bytes for free.
+
+Those are the reason we choose to store every set of records in a particular zone file as a single transaction.
```

Comment:
```suggestion
Those are the reasons we choose to store every set of records in a particular zone file as a single transaction.
```

I don't feel convinced that these are good reasons for doing this. Can we expand on this some more / write this more clearly? 

Really spell out the details or something. A structure like:
- Discussion on zone files, how big they are, what we're expecting to actually have to store
- Remind the reader that this probably seems infeasible to do initially only on Cardano
- Talk about Arweave / how much it costs
- Then conclude that with Arweave is a reasonable ish solution to the storage problem.


Then, afterwards the discussion on how we actually plan to use Arweave is great followup.

Reply:
This is outdated after the meeting, will update soon.